runtime/src/main/kotlin/generated/_ComparisonsNative.kt

```diff
@@ -136,6 +136,76 @@ public actual inline fun maxOf(a: Double, b: Double, c: Double): Double {
     return maxOf(a, maxOf(b, c))
 }
 
+/**
+ * Returns the greater of given values.
+ */
+@SinceKotlin("1.4")
+public actual fun <T : Comparable<T>> maxOf(a: T, vararg other: T): T {
```

Comment:
What about unsigned versions?

Reply:
Unsigned versions are in common source set.
See the PR for big Kotlin: https://upsource.jetbrains.com/kotlin/review/KOTLIN-CR-3810